management/src/main/java/io/micronaut/management/endpoint/env/EnvironmentEndpoint.java

```diff
@@ -17,104 +17,116 @@
 
```

Comment:
This is a breaking change as it is a modification to the public API.  Discussed with @jameskleeh and we decided we should be ok, as we couldn't see why someone would be creating one of these themselves, nor extending it for their own use.